composer.json

```diff
@@ -7,7 +7,13 @@
         "php": ">=5.4"
     },
     "php-ext": {
+        "extension-name": "timezonedb",
```

Comment:
Extension name is not strictly required, since the package name is already correct, but it may help to be explicit.